### PR TITLE
ssh-keys: cleanup

### PR DESCRIPTION
**Pull request by zimbatm**
* Remove unused keys
* Add missing members

### CODE REVIEW

File: ssh-keys.nix [hunk 1]
```diff
@@ -1,48 +1,39 @@
...
-    delroth
     hexa-gaia
     hexa-helix
     vcunat
     zimbatm
   ];
 
   infra = infra-core ++ [
+    _999eagle-laptop
+    _999eagle-tower
```
Comment (vcunat) [inline ssh-keys.nix:34]: These two are missing definition, right?

**Review by JulienMalka — DISMISSED** [review]
Adding people to `infra` is giving root access to `non-critical-infra` machines. Ideally, I'd not do that until people have been on-boarded to non-critical-infra and/or have started contributing to it. That being said, that's something I'd really want to see happen.

File: ssh-keys.nix (hunk 1 above)
Comment (JulienMalka) [reply ssh-keys.nix:34]: I do see a definition, after `zimbatm`

Comment (vcunat) [reply ssh-keys.nix:34]: Now I also see and find it.  I'm not sure what I did wrong when I searched last time.

**Comment by zimbatm** [comment]
Let me ask them.

**Comment by zimbatm** [comment]
Let me make a separate PR for the additions

**Review by mweinelt — APPROVED** [review]

**Pull request merged**